src/main/kotlin/org/oooc/kry/global/ErrorResponse.kt

```diff
@@ -2,30 +2,26 @@ package org.oooc.kry.global
 
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
-import org.springframework.util.MultiValueMap
 import java.util.*
 
-data class ErrorResponse(val status:HttpStatus
-                         , val error:String
-                         , val message:String
-                         , val timestamp: Date
-                         , val bindingErrors: List<String>) {
+data class ErrorBody(
+    val status:String,
+    val message:String,

```

Comment:
```suggestion
    val status: String,
    val message: String,
```
타입은 `val name: Type` 처럼 콜론 앞은 변수명에 붙이고 뒤에 한 칸을 띄어 씁니다.
상속은 `class ChildClass : ParentClass` 처럼 콜론 앞뒤로 한 칸을 띄어 씁니다.